src/fu-util.c

```diff
@@ -3386,6 +3386,54 @@ fu_util_sync_bkc(FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_util_prompt_complete(priv->completion_flags, TRUE, error);
 }
 
+static gboolean
+fu_util_security_modify_bios_attr(FuUtilPrivate *priv, FwupdSecurityAttr *attr, GError **error)
+{
+	g_autoptr(GString) body = g_string_new(NULL);
+	g_autoptr(GString) title = g_string_new(NULL);
+
+	g_string_append_printf(title,
+			       "%s: %s",
+			       /* TRANSLATORS: title prefix for the BIOS settings dialog */
+			       _("Configuration Change Suggested"),
+			       fwupd_security_attr_get_title(attr));
+
+	g_string_append(body, fwupd_security_attr_get_description(attr));
+	g_string_append(body, "\n\n");
+	g_string_append_printf(
+	    body,
+	    /* TRANSLATORS: the %1 and %2 are the values, e.g. "True" or "Windows10" */
+	    _("This tool can change the BIOS setting from '%s' to '%s' "
+	      "automatically, but it will only be active after restarting the "
+	      "computer."),
+	    fwupd_security_attr_get_bios_attr_current_value(attr),
+	    fwupd_security_attr_get_bios_attr_target_value(attr));
```

Comment:
I realized that we need to tell them _what_ BIOS setting we're changing.  The attribute title doesn't match the name of the string.

```suggestion
	    /* TRANSLATORS: the %1 is a BIOS setting name.  %2 and %3 are the values, e.g. "True" or "Windows10" */
	    _("This tool can change the BIOS setting '%s' from '%s' to '%s' "
	    "automatically, but it will only be active after restarting the "
	      "computer."),
	    fwupd_bios_attr_get_name(fwupd_security_attr_get_bios_attr_id(attr)),
	    fwupd_security_attr_get_bios_attr_current_value(attr),
	    fwupd_security_attr_get_bios_attr_target_value(attr));
```